tests/surrealdb/helpers/lifecycle.ts

```diff
@@ -0,0 +1,94 @@
+import { readFileSync } from 'fs';
+
+import { SessionType, TransactionType, TypeDB } from 'typedb-driver';
+import { v4 as uuidv4 } from 'uuid';
+
+import type { Provider } from '../../../src/index';
+import BormClient from '../../../src/index';
+// import { cloudConfig } from '../mocks/cloudConfig';
+import { testConfig } from '../mocks/testConfig';
+import { testSchema } from '../mocks/testSchema';
+import { Surreal } from 'surrealdb.node'
+import { customAlphabet } from 'nanoid'
+
+// NOTE SurrealDB does not accept hyphen, tried to use quotes still does not work
+const alphabet = '1234567890abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ'
+const nanoid = customAlphabet(alphabet,  21)
+
+const provider: Provider = 'surrealDB';
+
+const providerConfig = {
+	surrealDB: testConfig,
+};
+
+export const init = async () => {
```

Comment:
Instead of having this in the lifecycle, we will need to move this logic to the init() of the borm class, so it automatically inits every database. Then we can add some optional params like loading schemas etc depending on the databsse, this would reduce the redundancy so we can avoid having multiple lifecycle files. 

This as well can be done later when unifying folders